app/routes/search/agendaitems.js

```diff
@@ -170,4 +170,14 @@ export default class AgendaitemSearchRoute extends Route {
     });
     return true;
   }
+
+  postProcessPastAgendaVersions(entry) {
+    const pastAgendaitems = entry.agendaitemTreatment.agendaitems;
```

Comment:
Making agendaitemTreatment optional. Shouldn't happen on accept or prod, but in test this could be possible